pom.xml

```diff
@@ -106,7 +106,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>23.3-jre</version>
+            <version>27.0.1-jre</version>
```

Comment:
Bumping the Guava dependency is not strictly necessary. If for some reason you want to stay with 23.x, this is no problem. With the change in `KafkaHelper`, it will still be binary compatible with Guava 27.